pkg/cmd/dev/generate.go

```diff
@@ -323,3 +325,41 @@ import "C"
 
 	return nil
 }
+
+func (d *dev) generateJs(cmd *cobra.Command) error {
+	ctx := cmd.Context()
+
+	args := []string{
+		"build",
+		"//pkg/ui/workspaces/eslint-plugin-crdb:eslint-plugin-crdb",
+		"//pkg/ui/workspaces/db-console/src/js:crdb-protobuf-client",
+		"//pkg/ui/workspaces/cluster-ui:ts_project",
+	}
+	logCommand("bazel", args...)
+	if err := d.exec.CommandContextInheritingStdStreams(ctx, "bazel", args...); err != nil {
+		return fmt.Errorf("Building JS development prerequisites: %s", err.Error())
```

Comment:
This is throwing a lint error, but you should just be able to do

```
return fmt.Errorf("building JS development prerequisites: %w", err)
```

Note the sentence should start with a lowercase letter.

Reply:
How strange, I ran `./dev lint` and everything passed 0.o

Reply:
Ah sorry, that will only build `cockroach` for lints, not `dev`. `bazel build pkg/cmd/dev` will catch this.